gradle/java.gradle

```diff
@@ -11,7 +11,7 @@ compileJava {
 }
 
 task testJar(type: Jar, dependsOn: testClasses, group: "build") {
-    appendix = 'test'
+    classifier = 'test'
```

Comment:
note: i do not like the way how testMultiNode.sh expect dependencies - thinking about changing it to be passed from gradle itself.